src/Google/Client.hs

```diff
@@ -181,6 +226,47 @@ postGmailSend token email = do
     (postGmailSend' (pure . toBearer $ token) gmailSend)
     (mkClientEnv manager googleBaseUrl)
 
+getDriveFileList ::
+     Response.Token
+  -> Maybe Text
+  -> Maybe Text
+  -> IO (Either ServantError Response.FileList)
+getDriveFileList token query orderBy = do
+  manager <- newManager tlsManagerSettings
+  runClientM
+    (getDriveFileList'
+       (pure . toBearer $ token)
+       query
+       orderBy)
+    (mkClientEnv manager googleBaseUrl)
+
+createDriveFileMultipart ::
+     Response.Token
+  -> Form.MultipartBody
+  -> IO (Either ServantError Response.FileResource)
+createDriveFileMultipart token body = do
+  manager <- newManager tlsManagerSettings
+  runClientM
+    (createDriveFileMultipart'
+      (pure . toBearer $ token)
+      (Just "multipart")
+      body)
+    (mkClientEnv manager googleBaseUrl)
+
+exportDriveFile ::
+     Response.Token
+  -> Text
+  -> Text
+  -> IO (Either ServantError BS.ByteString)
+exportDriveFile token fileId mimeType = do
```

Comment:
I would like to make `mimeType` and `fileId` type safe.
Is it possible to create new type `MimeType` and `FileId`?

The `query` and `orderBy` used elsewhere should also be type safe, but if it's too labor intensive, you can just leave it.